test/fixtures/pkg/kubectl/cmd/apply/tpr-entry.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "unit-test.test.com/v1"
+kind: TestTpr
```

Comment:
again, not TPR-specific... this is testing the ability to apply generic objects not compiled into kubectl